pkg/constants/constants.go

```diff
@@ -33,12 +33,15 @@ const (
 	DefaultKnativeServingNamespace      = "knative-serving"
 	DefaultKnativeServingFeaturesCMName = "config-features"
 
+	DefaultKedaInterceptorProxyName = "keda-add-ons-http-interceptor-proxy"
```

Comment:
The upstream keda http interceptor proxy is called `keda-add-ons-http-interceptor-proxy`?
`keda-addons-http-interceptor-proxy` sounds more reasonable

Reply:
Yes, that's exactly the name of that Service, and no need to change.